### DISCUSSION TITLE
How do you send the "Off with effect (0x40)" command and payload to a Light?

### DISCUSSION THREAD

**Discussion by rechichidaniel** [Koenkk/zigbee2mqtt#15765] [General]
How can I make Z2M send the `Off with effect (0x40)` command to a light? The payload would need to include the `Delayed All Off (0x00)` effect identifier and the `Fade to off in 0.8 seconds (0x00)` effect variant.

I'm trying to get my Z2M/CC2652P2 setup to send the same command as my Hue Bridge when turning lights off. Below is what Wireshark+nrf52840 says Hue is sending.

```
ZigBee Cluster Library Frame
    Frame Control Field: Cluster-specific (0x11)
        .... ..01 = Frame Type: Cluster-specific (0x1)
        .... .0.. = Manufacturer Specific: False
        .... 0... = Direction: Client to Server
        ...1 .... = Disable Default Response: True
    Sequence Number: 59
    Command: Off with effect (0x40)
    Payload
        Effect Identifier: Delayed All Off (0x00)
        Effect Variant: Fade to off in 0.8 seconds (0x00)
```

**Comment by rechichidaniel** [comment]
Just jumping in to answer my own question. This is how.

**Cluster**: `0x0006` which I pulled from the Hue packet.
**Command**: `0x40` which I also pulled from the Hue packet.
**Payload**:
```json
{
"effectid":0,
"effectvariant":0
}
```
![image](https://user-images.githubusercontent.com/21980408/209769893-998a84ba-b343-4662-8f6e-7ea6a692ffa7.png)

**Reply by andrei-lazarov** [reply]
How did you find out what the fields are called? (`effectid`, `effectvariant`)
